test/unit/controllers/mainCtrl.spec.js

```diff
@@ -48,13 +61,17 @@ describe('app.config,js', function ()
         {
             expect(mainCtrl.arrayCurrency).toBe('actual table of currency');
```

Comment:
I see `expect` twice :P for one `it` described as `should message be set to text`